### PR TITLE
Add documentation for advanced diagnostics

### PR DESCRIPTION
**Pull request by peterschen**
This PR adds documentation on how to enable advanced diagnostics (taking a pcapng capture from the VM joining AD) for troubleshooting/debugging purposes.

### CODE REVIEW

**Review by jpassing — APPROVED** [review]

File: ad-joining/README.md [hunk 1]
```diff
@@ -46,6 +48,14 @@ Cloud Run environment:
 
 * `GOOGLE_APPLICATION_CREDENTIALS` (required for testing): Path to a service account key of a service account that permits the app to interact with Compute Engine. 
 
+### Advanced diagnostics
+
+The server side log will provide details on server operations like creating/modiying computer accounts. In some cases additional diagnostics are required. By enabling advanced diagnostics, a network capture will be taken from the join process on the VM that is joining Active Directory and stored to Google Cloud Storage. The storage bucket needs to be created before enabling advanced diagnostics and the VMs' service account needs to have write access to it. Network captures will be stored in the `captures` subdirectory in [PCAP Next Generation (pcapng) Capture file format](https://www.ietf.org/staging/draft-tuexen-opsawg-pcapng-02.html).
```
Comment (idofl) [inline ad-joining/README.md:53]: modiying -> modifying

**Review by idofl — CHANGES_REQUESTED** [review]
LGTM except for one typo

File: ad-joining/README.md (hunk 1 above)
Comment (peterschen) [reply ad-joining/README.md]: Fixed.

**Review by idofl — COMMENTED** [review]
LGTM

**Pull request merged**